src/modals/zaken/ZaakForm.vue

```diff
@@ -0,0 +1,239 @@
+<script setup>
+import { navigationStore, zaakStore, zaakTypeStore } from '../../store/store.js'
+</script>
+
+<template>
+	<NcModal ref="modalRef"
+		label-id="zaakForm"
+		@close="closeModal">
+		<div class="modalContent">
+			<h2>Zaak {{ zaakStore.zaakItem?.id ? 'aanpassen' : 'aanmaken' }}</h2>
+
+			<div v-if="success !== null">
+				<NcNoteCard v-if="success" type="success">
+					<p>Zaak succesvol {{ zaak.id ? 'aangepast' : 'aangemaakt' }}</p>
+				</NcNoteCard>
+				<NcNoteCard v-if="!success" type="error">
+					<p>Zaak niet succesvol {{ zaak.id ? 'aangepast' : 'aangemaakt' }}</p>
+				</NcNoteCard>
+				<NcNoteCard v-if="error" type="error">
+					<p>{{ error }}</p>
+				</NcNoteCard>
+			</div>
+
+			<div v-if="success === null" class="form-group">
+				<NcTextField :disabled="zaakLoading"
+					label="Identificatie"
+					maxlength="255"
+					:value.sync="zaak.identificatie"
+					required />
+				<NcTextField :disabled="zaakLoading"
+					label="Omschrijving"
+					maxlength="255"
+					:value.sync="zaak.omschrijving" />
+				<NcTextField :disabled="zaakLoading"
+					label="Bronorganisatie"
+					maxlength="9"
+					:value.sync="zaak.bronorganisatie"
+					required />
+				<NcTextField :disabled="zaakLoading"
+					label="VerantwoordelijkeOrganisatie"
+					maxlength="9"
+					:value.sync="zaak.verantwoordelijkeOrganisatie"
+					required />
+				<NcTextField :disabled="zaakLoading"
+					label="Startdatum"
+					maxlength="9"
+					:value.sync="zaak.startdatum"
+					required />
+				<NcSelect v-bind="zaakType"
+					v-model="zaakType.value"
+					input-label="Zaaktype"
+					:loading="zaakTypeLoading"
+					:disabled="zaakLoading || zaakTypeLoading"
+					required />
+				<NcSelect v-bind="archiefstatus"
+					v-model="archiefstatus.value"
+					input-label="Archiefstatus"
+					:disabled="zaakLoading"
+					required />
+				<NcTextField :disabled="zaakLoading"
+					label="Registratiedatum"
+					maxlength="255"
+					:value.sync="zaak.registratiedatum" />
+				<NcTextArea :disabled="zaakLoading"
+					label="Toelichting"
+					:value.sync="zaak.toelichting" />
+			</div>
+
+			<NcButton v-if="success === null"
+				:disabled="loading || !zaak.identificatie || !zaakTypeLoading || !zaak.bronorganisatie || !zaak.verantwoordelijkeOrganisatie || !zaak.startdatum"
```

Comment:
This will always be disabled because of `!zaakTypeLoading` this should be without the !